src/components/Searchbar/Search.tsx

```diff
@@ -17,6 +18,14 @@ export const Search = () => {
     staleTime: 1000 * 60 * 60 // 60 minutes caching
   });
 
+  const { data: newTitles } = useQuery({
+    queryKey: ['titles'],
+    queryFn: async () => {
+      return TitleService.getAllTitles();
+    },
+    staleTime: 1000 * 60 * 60 // 60 minutes caching
+  });
+
   const [posts, setPosts] = React.useState<IJsonAlbum[]>([]);
```

Comment:
remove extra state for posts, output data from titles
remove mannual search by title using regex, we will use api for search